README.md

```diff
@@ -1,2 +1,146 @@
 # aedes-server-factory
-Aedes server factory. Supports tcp, tls, http, https, http2, ws, wss and proxy decoders
+
+Aedes server factory. Supports tcp, http, http2, ws, and proxy decoders
+
+Work In Progress: tls, https, http2, wss
+
+## Install
+
+To install aedes-server-factory :
+
+```sh
+npm install aedes-server-factory
+```
+
+# Options
```

Comment:
I would make it more clear that only one server can be created for time. I mean that maybe a user could think that by setting multiple servers protocols to true it returns all of them (we could add this support too)? 

Also you are missing tls secure options (key, cert etc)

Reply:
Example in aedes-cli: 

https://github.com/moscajs/aedes-cli/blob/master/lib/cli.js#L204
https://github.com/moscajs/aedes-cli/blob/master/lib/cli.js#L417

Reply:
I'm hesitating for the `ws` `http` `https` `tls`, as object instead of boolean, since there is probably redundant options for https and tls.
But i agree for the potential confusions regarding the number of server generated, so i will fix the doc.
Anyway for this PR i'd like to focus on properly handling ws and tcp, with proxy / http header / stream.
When this one is merged, we can take time to think on how to improve the options, handle certificates / keys and eventually create multiple server, would that be ok ?

Reply:
It's ok for me no problems :)